File: OneDrive/Documentos/Store_Herencia/Store_Herencia/src/store/herencia/Householde_Appliance.java
package store.herencia;

public class Householde_Appliance extends Product {
    private String model;
    public Householde_Appliance(String name, double price, String importDate, int stock, String model){
        super(name, price, importDate, stock);
        this.model = model;
    }
    
    @Override
    public String toString() {
        return super.getData() + String.format(", model: %s", model);
    }
}

File: PycharmProjects/Store_Herencia/py/Householde_Appliance.py
from Product import Product

class Householde_Appliance(Product):
    def __init__(self, name, price, import_date, stock, model):
        super().__init__(name, price, import_date, stock)
        self.model = model

    def __str__(self):
        return super().get_data() + f"Model: {self.model}"